nmrfx-core/src/main/java/org/nmrfx/datasets/DatasetParameterFile.java

```diff
@@ -55,18 +55,10 @@ public String getParameterFileName() {
     }
 
     public static String getParameterFileName(String fileName) {
-        int len = fileName.length();
-        String parFileName;
-        int extLen = 0;
-        if (fileName.endsWith(".nv")) {
-            extLen = 3;
-        } else if (fileName.endsWith(".ucsf")) {
-            extLen = 5;
-        } else if (fileName.endsWith(".nvlnk")) {
-            extLen = 6;
-        }
-        parFileName = fileName.substring(0, len - extLen) + ".par";
-        return parFileName;
+        Pattern pattern = Pattern.compile(".(nv|ucsf|nvlnk)$");
```

Comment:
Shouldn't the pattern be ``\\.(nv|ucsf|nvlnk)$`` to detect the point?
If not a filename such as ``myfile_nvlnk`` become also ``myfile.par`` and ``myfilenvlnk`` become ``myfil.par``

Reply:
Yes, I forgot to escape it, changed this one and the same error in ProjectBase.java